Description: Fix requirements.txt
 This avoids FTBFS with dh_gencontrol being confused.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2016-06-09

Index: deb-glance/requirements.txt
===================================================================
--- deb-glance.orig/requirements.txt
+++ deb-glance/requirements.txt
@@ -8,8 +8,7 @@ pbr>=1.6 # Apache-2.0
 SQLAlchemy<1.1.0,>=1.0.10 # MIT
 eventlet!=0.18.3,>=0.18.2 # MIT
 PasteDeploy>=1.5.0 # MIT
-Routes!=2.0,!=2.1,!=2.3.0,>=1.12.3;python_version=='2.7' # MIT
-Routes!=2.0,!=2.3.0,>=1.12.3;python_version!='2.7' # MIT
+Routes
 WebOb>=1.2.3 # MIT
 sqlalchemy-migrate>=0.9.6 # Apache-2.0
 httplib2>=0.7.5 # MIT
@@ -54,7 +53,7 @@ glance-store>=0.18.0 # Apache-2.0
 semantic-version>=2.3.1 # BSD
 
 debtcollector>=1.2.0 # Apache-2.0
-cryptography!=1.3.0,>=1.0 # BSD/Apache-2.0
+cryptography
 cursive>=0.1.1 # Apache-2.0
 
 # timeutils
